### PR TITLE
Add test that checks if all values of the answer charts are defined i…

### PR DESCRIPTION
**Pull request by olafveerman**
…n chart-values.csv

### CODE REVIEW

File: input/charts.csv [hunk 1]
```diff
@@ -7,7 +7,7 @@ upcomingAuctions,answer,Upcoming auctions,What is the volume of auctions announc
 wholesaleDistortions,answer,Wholesale price distortions,Are fossil fuels generators benefiting from subsidies that affect the price at which they can sell electricity onto the wholesale market?,fundamentals,Wholesale price distortions,,,
 sectorWholesale,answer,Power sector wholesale,,fundamentals,Wholesale power market,,,
 currencyVariation,answer,Currency variation,How high is the ten year variation of the local currency against the US dollar?,fundamentals,Currency variation,,,
-offtakerRisk,answer,Offtaker risk,How high is the offtake risk for independent generators?,fundamentals,Offtaker risk,,,
+offtakerRisk,range,Offtaker risk,How high is the offtake risk for independent generators?,fundamentals,Offtaker risk,,,
```
Comment (danielfdsilva) [inline input/charts.csv]: The type was changed, but the options were not.
They should be:
```
Offtaker risk,2,Very High
Offtaker risk,0,Very Low
```

File: scripts/generate-data.js [hunk 2]
```diff
@@ -76,10 +76,11 @@ function generateProfileData (geo, indicators) {
 // Extract the section copy for all sections from the 'note' property.
 function generateSectionCopy (geo, indicators) {
   let countryCopy = indicators
+    // The .find checks if value of a particular year is not null
     .filter(i =>
       i.geography === geo.name &&
       i.category === 'BNEF Take' &&
-      i.note !== ''
+      i.values.find(v => v.year === 2017).value
```
[the remark this answers is not in the document]
Comment (olafveerman) [reply scripts/generate-data.js]: Ticketing for later

[the remark this answers is not in the document]
Comment (olafveerman) [reply scripts/generate-data.js]: https://github.com/climatescope/cs-datapipeline/issues/34

File: input/charts.csv (hunk 1 above)
Comment (olafveerman) [reply input/charts.csv]: The scale is not linear, so keeping the current steps to see what we have to do here.

Comment (danielfdsilva) [reply input/charts.csv]: Ok, let's get feedback

**Pull request merged**